content/news/playtest-20150524.md

```diff
@@ -0,0 +1,57 @@
+It has been a [month of busy development](https://github.com/OpenRA/OpenRA/pulse/monthly). Time to get back to testing!
+
+The major changes this time around include:
+
+  - A whole new in-game map editor, with many benefits over the old editor.
+  - Improved system integration with `openra://` server join URLs. Give it a try from our [live server list](/games/)!
+  - Implemented production building upgrades that unlock additional technology for Dune 2000.
+  - Several enhanced classic missions have been added to the Tiberian Dawn mod.
+  - In the Red Alert and D2k mods you will now be notified upon encounter of enemy units.
+
+Unfortunately, an oversight in the way that we package our windows installer meant that the performance of our Windows release-20150429 builds was much worse than it should have been. We have fixed this for playtest-20150524, so Windows players should find the game runs about 30% faster!<br>
+Also general performance improvements have been integrated that affect all platforms. Kudos to [RoosterDragon](https://github.com/RoosterDragon) for his continued effort to optimize our source code.
+
+The refined system integration adds a lot of convenient features:
+
+  - The hardware cursors will now work more reliably on Microsoft Windows.
+  - On Mac the dialogs won't hijack finder and the dock icon won't get lost upon restarting.
+  - Linux AppStream compatible software centers will now show a nice preview.
+  - Double-clicking an `.orarep` file will now directly start playback of the replay.
+
+Our new in-game map editor took over a year of development time and it was worth it.
+
+  - It is truly cross-platform as it uses our game UI; finally a map editor on Mac OS X!
+  - No more rendering glitches. Almost WYSIWYG editing, because we re-use the game renderer.
+  - Support for all file formats, i.e. voxels, isometric terrain and Dune 2000 sprite actors.
+  - You can now keep an overview of installed custom maps and delete them in the same menu.
+
+Click `Extras` - `Map Editor` from the main menu in any mod. We will remove the legacy `OpenRA.Editor.exe` in the nearby future.
```

Comment:
Why don't you want to included my suggested request for our enthusiastic mapping community to give feedback on the new editor?